### PR TITLE
fix:non-uniformity of icons used

### LINKED ISSUE #286: fix non-uniformity of icons used
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**
`admin/events`

screenshot 7

`events/<event-id>/tickets/discount-codes`
screenshot 8

**Expected behavior:**

Same icons must be used across the project for the same functionality

I would like to work on this

### PR DESCRIPTION
**Pull request by utkarshnath**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

Fixes #286
@CosmicCoder96 @geekyd @anu-007 @sumedh123 @harshitagupta30  plz review
https://hidden-badlands-35601.herokuapp.com

### CODE REVIEW

File: app/templates/admin/permissions/system-roles.hbs [hunk 1]
```diff
@@ -46,10 +46,10 @@
       </td>
       <td>
         <div class="ui horizontal compact basic buttons">
-          {{#ui-popup content=(t 'Edit') class='ui icon button' position='left center'}}
+          {{#ui-popup content=(t 'Edit') class='ui icon button'}}
             <i class="edit icon"></i>
           {{/ui-popup}}
-          {{#ui-popup content=(t 'Delete') class='ui icon button' position='left center'}}
```
Comment (CosmicCoder96) [inline app/templates/admin/permissions/system-roles.hbs]: The PR #240 by @sumedh123 did the exact opposite of these changes, I don't think this is required.

File: app/templates/admin/permissions/system-roles.hbs [hunk 2]
```diff
@@ -46,10 +46,10 @@
       </td>
       <td>
         <div class="ui horizontal compact basic buttons">
-          {{#ui-popup content=(t 'Edit') class='ui icon button' position='left center'}}
```
Comment (CosmicCoder96) [inline app/templates/admin/permissions/system-roles.hbs]: The PR #240 by @sumedh123 did the exact opposite of these changes, I don't think this is required.

File: app/templates/admin/permissions/system-roles.hbs [hunk 3]
```diff
@@ -64,10 +64,10 @@
...
```
Comment (CosmicCoder96) [inline app/templates/admin/permissions/system-roles.hbs]: same here

File: app/templates/admin/permissions/system-roles.hbs [hunk 4]
```diff
@@ -64,10 +64,10 @@
...
+          {{#ui-popup content=(t 'Edit') class='ui icon button'}}
             <i class="edit icon"></i>
           {{/ui-popup}}
-          {{#ui-popup content=(t 'Delete') class='ui icon button' position='left center'}}
```
Comment (CosmicCoder96) [inline app/templates/admin/permissions/system-roles.hbs]: same here

File: app/templates/admin/permissions/system-roles.hbs (hunk 1 above)
Comment (utkarshnath) [inline app/templates/admin/permissions/system-roles.hbs]: @CosmicCoder96 `left center` should have been done only for vertical menu. As here the menu is horizontal `left center` hides the other option in same menu. That's why I removed it here
screenshot 4

Comment (CosmicCoder96) [inline app/templates/admin/permissions/system-roles.hbs]: @utkarshnath That makes sense, should have had been noticed in PR #240 itself, approving these then.

Comment (geekyd) [inline app/templates/admin/permissions/system-roles.hbs]: @utkarshnath Thanks for explaining :+1:

**Pull request merged**